installers/sirepo-dev/radiasoft-download.sh

```diff
@@ -36,11 +36,11 @@ sirepo_dev_main() {
     done
     cd sirepo
     install_yum_install nodejs
-    for p in jshint karma karma-jasmine karma-phantomjs-launcher jasmine-core; do
-        if ! npm list "$p" >& /dev/null; then
-           npm install "$p" >& /dev/null
-        fi
-    done
+    npm install
+    install_tmp_dir
+    install_download https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm > chrome.rpm
```

Comment:
I think you can put this on one line, and then yum knows how to manage it:

```txt
install_yum_install https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm
```

You don't need install_tmp_dir then or even cd -

Reply:
https://github.com/radiasoft/download/pull/115/commits/775fe9111be31abe3b18ecc1d47f558681eccd19 . I changed perl-dev too which is where I copied the code from.